internal/generate/kubernetes/generate.go

```diff
@@ -0,0 +1,101 @@
+package kubernetes
+
+import (
+	"bytes"
+	"github.com/commitdev/commit0/internal/config"
+	"github.com/commitdev/commit0/internal/templator"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+)
+
+func Generate(templator *templator.Templator, config *config.Commit0Config, wg *sync.WaitGroup) {
+	templator.Kubernetes.TemplateFiles(config, false, wg)
+
+	if config.Kubernetes.Deploy {
+		_tf_init := tf_init()
+		_tf_plan := tf_plan()
+		execute(_tf_init)
+		execute(_tf_plan)
+	}
+
+}
+
+// Terraform init cmd
+func tf_init() *exec.Cmd {
+
+	return exec.Command("terraform", "init")
+}
+
+// Terraform plan cmd
+func tf_plan() *exec.Cmd {
+
+	return exec.Command("terraform", "plan")
+}
+
+// Executes cmd passed in
+func execute(cmd *exec.Cmd) {
+	dir, err1 := filepath.Abs(filepath.Dir(os.Args[0]))
+	if err1 != nil {
+		log.Fatal(err1)
+	}
+
+	cmd.Dir = dir + "/kubernetes/terraform/environments/staging"
+
+	var errStdout, errStderr error
+	stdoutIn, _ := cmd.StdoutPipe()
+	stderrIn, _ := cmd.StderrPipe()
+	stdout := NewCapturingPassThroughWriter(os.Stdout)
```

Comment:
What is all this stuff for? You should be able to get the above stdoutpipe and then just call `go io.Copy(os.Stdout, stdoutIn)`